Sources/CardPayments/CardClient.swift

```diff
@@ -113,19 +113,19 @@ public class CardClient: NSObject {
                         return
                     }
                 
-                    analyticsService?.sendEvent("card-payments:3ds:confirm-payment-source:challenge-required")
+                    analyticsService?.sendEvent("card-payments:confirm-payment-source:challenge-required")
```

Comment:
On Android we do have every event related to `approveOrder()` named with `approve-order`. Here's an example in [CardAnalytics.kt](https://github.com/paypal/paypal-android/blob/main/CardPayments/src/main/java/com/paypal/android/cardpayments/CardAnalytics.kt).

Reply:
Yes, I had opened a PR to rename these events but closed them and wanted to just clean up things that were mislabeled as "3ds" in this PR. I wanted to make changes until we have alignment in iOS/Android on analytic events.